openmls/src/group/tests/test_group.rs

```diff
@@ -408,14 +410,15 @@ fn group_operations() {
             FramedContentBodyIn::Application(message) if message.as_slice() == &message_alice[..]));
 
     // === Bob updates and commits ===
-    let bob_new_leaf_node = group_bob
-        .own_leaf_node()
-        .unwrap()
-        .updated(
+    let mut bob_new_leaf_node = group_bob.own_leaf_node().unwrap().clone();
+    bob_new_leaf_node
+        .update(
             ciphersuite,
-            TreeInfoTbs::Update(group_bob.own_tree_position()),
             provider,
-            &alice_credential_with_keys.signer,
+            &bob_credential_with_keys.signer,
```

Comment:
I don't think this produced an invalid signature, it rather converted the identity from Bob to Alice. Since we put the proposal in the ProposalStore directly, we bypass all validation checks we have. Feel free to convert this into an issue, however.